server/js/sockets.js

```diff
@@ -7,14 +7,47 @@ const sockets = {};
 const mqtt = require('mqtt');
 const os = require('os');
 
-function publicMqttConnected() {
-  console.log('Connected to public mqtt broker');
+// Public MQTT broker
+let PUBLISH_ONLINE = false;
```

Comment:
me likey this

Reply:
Would've prevented the problem I was having on my own laptop too! Also useful cos the DAS only has internet for 2 weeks of the year :)